sdk/go/bootstrap.go

```diff
@@ -61,7 +61,7 @@ func (sdk mfSDK) AddBootstrap(key string, cfg BoostrapConfig) (string, error) {
 		if err := encodeError(resp.StatusCode); err != nil {
 			return "", err
```

Comment:
What if there is an error we don't need to wrap anything? I am not sure how SDK error looks like

Reply:
When there is an HTTP error with a mainflux equivalent we only use the custom error. Otherwise we wrap the HTTP message with the custom mainflux error for this kind of function.